src/components/ContentSidebar/ContentSidebar.js

```diff
@@ -434,6 +464,17 @@ class ContentSidebar extends PureComponent<Props, State> {
         this.setState({ accessStats, accessStatsError: undefined });
     };
 
+    /**
+     * User fetch success callback
+     *
+     * @private
+     * @param {Object} currentUser - User info object
+     * @return {void}
+     */
+    fetchCurrentUserSuccessCallback = (currentUser: User): void => {
+        this.setState({ currentUser, currentUserError: undefined });
```

Comment:
Does the User flow type match the API response? I thought we were going to format from snake_case to camelCase?

Reply:
As discussed in scrum, this is for another PR/task. I'll format it before passing to `setState()`

Reply:
I would say don't set the state then right? We want the state item to be of our type, not from the API